### PR TITLE
LM-134: Add file size metrics

### PR DESCRIPTION
**Pull request by jeohist**
In Mendix 7.4 we added a `size` column to `system$filedocument`
which allows us to get an overview of the file storage usage:
<https://docs.mendix.com/releasenotes/studio-pro/7.4#other-improvements>
This commit outputs these metrics for compatible versions.

### CODE REVIEW

**Comment by joshp123** [comment]
This may have product considerations, let's verify this + get sign-off from them before merging.

**Comment by tomedegroot** [comment]
Hi Joey,

Great that you take the initiative. I guess we need to work on the CloudPortal side as wel to actually display the graphs to users?

File: buildpack/telemetry/metrics.py
Comment (udh-mendix) [inline buildpack/telemetry/metrics.py:496]: why remove cursor.execute from the try block?

Comment (jeohist) [reply buildpack/telemetry/metrics.py:496]: It's not removed, the try block is just moved to L353, to be consistent with the way we get the number of files. The old logic essentially went "try to get size of files, if it fails, must be the version!". In this way it first checks the version, and then if it fails it'll throw an actual exception.

**Review by udh-mendix — APPROVED** [review]
LGTM

**Pull request merged**
